connectors/utils.py

```diff
@@ -405,3 +405,77 @@ def get_event_loop():
             loop = asyncio.new_event_loop()
             asyncio.set_event_loop(loop)
     return loop
+
+
+DEFAULT_PAGE_SIZE = 100
+
+
+class ESIndex(ESClient):
```

Comment:
`utils` is usually where we put stuff we don't know where to put else where until they grow into something more substantial and meaningful -- like a few functions and classes.

Since we starting to have a couple of classes for Elasticsearch interactions, I think it would make sense to move them to `es.py` -- not `elasticsearch.py` to avoid shadowing the elasticseatch library that already uses that namespace.

Reply:
Yep, it makes to me 👍🏻